pegasus/sites.v3/code.org/views/codebytes_skinnybanner.haml

```diff
@@ -1,18 +0,0 @@
-.breakoutquote{style:"padding: 0 18px; border-radius: 15px; background-image: linear-gradient(to top left, #a6e3e8, #d9f3f5)"}
```

Comment:
would it be worth keeping this in case we want to easily resurrect it next year? Or do we think that is unlikely?

Reply:
Yeah, I debated this - I bet there is a chance we'll want it again. thanks!